<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COaDialog</class>
 <widget class="QDialog" name="COaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COaDialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>朕准了</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePwd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>72</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>年份：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>102</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>季度：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboYear">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboQuarter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>第1季度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>第2季度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>第3季度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>第4季度</string>
    </property>
   </item>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoDesign">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>策划</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>部门：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoProgram">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>程序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoQa">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
